src/main/java/org/sagebionetworks/web/client/jsinterop/SqlDefinedTableEditorModalProps.java

```diff
@@ -0,0 +1,49 @@
+package org.sagebionetworks.web.client.jsinterop;
+
+import jsinterop.annotations.JsFunction;
+import jsinterop.annotations.JsNullable;
+import jsinterop.annotations.JsOverlay;
+import jsinterop.annotations.JsPackage;
+import jsinterop.annotations.JsType;
+
+@JsType(isNative = true, namespace = JsPackage.GLOBAL, name = "Object")
+public class SqlDefinedTableEditorModalProps extends ReactComponentProps {
+
+  // going to match the props sent from src
+  @FunctionalInterface
+  @JsFunction
+  public interface OnUpdate {
+    void OnUpdate();
```

Comment:
camelCase for consistency. You could also just define one `Callback` interface for both onUpdate and onCancel, but that's up to you
```suggestion
    void onUpdate();
```